apps/samradsgatt/graphql/getAllCases.ts

```diff
@@ -0,0 +1,9 @@
+import gql from 'graphql-tag'
+
+export const GET_ALL_CASES = gql`
```

Comment:
The best practice (with near-operation-file) is to define graphql operations close to where they're used (eg the page), instead of in a shared "graphql" folder. Note that by default you can't have non-page files in the pages folder. Two ways to fix this:

1. Move the actual page component to a screens subfolder (like we do in the web project). The pages folder has small stubs which re-export stuff from the screens folder.

2. Configure [`pageExtensions`](https://nextjs.org/docs/api-reference/next.config.js/custom-page-extensions#including-non-page-files-in-the-pages-directory). That way you can have `pages/about.page.tsx`, `pages/about.graphql` and `pages/about.generated.ts`, and only the `page.tsx` file is considered an actual page.

Reply:
Will fix that in a seperate pull request. That query was mostly for testing purposes :)